src/sidebar/store/modules/annotations.js

```diff
@@ -485,13 +509,15 @@ export default {
   },
 
   selectors: {
-    annotationExists,
-    noteCount,
     annotationCount,
-    orphanCount,
-    isWaitingToAnchorAnnotations,
+    annotationExists,
     findAnnotationByID,
     findIDsForTags,
+    isWaitingToAnchorAnnotations,
```

Comment:
This module is getting long so I took it upon myself to alphabetize its exports. Maybe silly.